300-mint.md

```diff
@@ -0,0 +1,266 @@
+300-inscriptions/
+```
+```
+Inscribing 300-inscriptions/1.json at c1c9dbf3b86ea75a5a4fa80cf7aef14c2840bc92c254f11188e656198f35f892:0:7519
+{
+  "commit": "72cc2b074f19c37ca0f81e2ad9177abb8f918c42a2e8d6e16314ee8f69af8638",
+  "inscription": "d4f79e33497fcb263274916d9fef0b105107a588f7b823e29babda435cf4bf27i0",
+  "parent": "db3b817f1676b7b00f6ed90dea14821b60ffc4f995af156b8b2c0f8ed5bdb829i0",
+  "reveal": "d4f79e33497fcb263274916d9fef0b105107a588f7b823e29babda435cf4bf27",
+  "fees": 682
+}
+Inscribing 300-inscriptions/2.json at 72cc2b074f19c37ca0f81e2ad9177abb8f918c42a2e8d6e16314ee8f69af8638:0:7518
+{
+  "commit": "21d63c5491e5ea1a9598acb980d8fcf538ca4b7642ad8503ca8f9acea31f55a8",
+  "inscription": "5c0f07b2aa7bf17e8528d289034210a5b6fbee9d39b043d7b9632a5a9cc3064ai0",
+  "parent": "db3b817f1676b7b00f6ed90dea14821b60ffc4f995af156b8b2c0f8ed5bdb829i0",
+  "reveal": "5c0f07b2aa7bf17e8528d289034210a5b6fbee9d39b043d7b9632a5a9cc3064a",
+  "fees": 769
+}
+
+... [TRUNCATED] 
+
+Inscribing 300-inscriptions/9.json at cdaca6dd542e14a9ef29cc1d7567e07928fd5782342b92d1e47955a353d24b37:0:7511
+{
+  "commit": "616f407f07c3ee2361f9cee0243219eae8dfb945b38e3dae1393c189624cb5f1",
+  "inscription": "66311ed501c4ace9628e9b183b10633fb62cdd518a41dcfdd299ce4feb6c3542i0",
+  "parent": "db3b817f1676b7b00f6ed90dea14821b60ffc4f995af156b8b2c0f8ed5bdb829i0",
+  "reveal": "66311ed501c4ace9628e9b183b10633fb62cdd518a41dcfdd299ce4feb6c3542",
+  "fees": 769
+}
+Inscribing 300-inscriptions/10.json at 616f407f07c3ee2361f9cee0243219eae8dfb945b38e3dae1393c189624cb5f1:0:7510
+{
+  "commit": "0ba61faa20a556f6e22a23b3c43a002395049133ce2586bbc9a06ad9fc7221ff",
+  "inscription": "c49d89f52b7209b3a5fb3ab4d826c6b29a99972718ba677008460651e2ebcd38i0",
+  "parent": "db3b817f1676b7b00f6ed90dea14821b60ffc4f995af156b8b2c0f8ed5bdb829i0",
+  "reveal": "c49d89f52b7209b3a5fb3ab4d826c6b29a99972718ba677008460651e2ebcd38",
+  "fees": 770
+}
+
+Success!
+10 new inscriptions pending in the mempool.
+
+To continue inscribing, wait for the block to be mined and run:
+ord wallet inscribe-chain --fee-rate 1.5 --parent db3b817f1676b7b00f6ed90dea14821b60ffc4f995af156b8b2c0f8ed5bdb829i0 --destination tb1p2denmwlt3hkdnjdamm399dp6d36y3a53fsa0vaj8fxfn6560m6sq8m0glg --satpoint 0ba61faa20a556f6e22a23b3c43a002395049133ce2586bbc9a06ad9fc7221ff:0:7509 300-inscriptions/
+```
+
+4.10 Wait for the previous transaction chain to be mined  
+4.11 Run the next `inscribe-chain` command
+```
+ord12 wallet inscribe-chain --fee-rate 1.5 --parent db3b817f1676b7b00f6ed90dea14821b60ffc4f995af156b8b2c0f8ed5bdb829i0 --destination tb1p2denmwlt3hkdnjdamm399dp6d36y3a53fsa0vaj8fxfn6560m6sq8m0glg --satpoint 0ba61faa20a556f6e22a23b3c43a002395049133ce2586bbc9a06ad9fc7221ff:0:7509 300-inscriptions/
+```
+4.12 Loop through steps `4.10` and `4.11` until the folder `300-inscriptions` has no more files left to be inscribed.
+
+## 5. Reveal
+5.1 Broadcast the commit transaction from step `3.3`
+```
```

Comment:
can't we do this first, back just before 3.5?

Reply:
either way should work